int64 ag_id
int64 tg_id
int64 leader_id
int64 help
int64 timestep
int64[] ag_involved
int64 intended_recipient